src/uucore/Cargo.toml

```diff
@@ -30,6 +30,8 @@ dunce = { version = "1.0.4", optional = true }
 wild = "2.2.1"
 glob = { workspace = true, optional = true }
 iana-time-zone = { workspace = true, optional = true }
+# TODO: Set optional to true, activated by feature i18n
+icu = { workspace = true, optional = false, features = ["compiled_data"] }
```

Comment:
i asked myself the same question about features.
is there a world where we don't want to have i18n for these ? (maybe?)

Reply:
I don't think we would need an `icu` dependency if we don't need `i18n`
Though for now, I've just not set the feature because it was breaking my IDE :sweat_smile: